tests/system/small/test_anywidget.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import math
+
+import pytest
+
+import bigframes as bf
+
+
+def test_repr_anywidget_initial_state(
+    penguins_df_default_index: bf.dataframe.DataFrame,
+):
+    pytest.importorskip("anywidget")
```

Comment:
Nit: Since all tests in this file require anywidget, let's move the importorskip to the module level.

I think this should work.

```suggestion


pytest.importorskip("anywidget")


def test_repr_anywidget_initial_state(
    penguins_df_default_index: bf.dataframe.DataFrame,
):
```

Same for the other functions in this file.